pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -131,7 +131,7 @@ func doUpdate(ctx *zedmanagerContext,
 		return changed
 	}
 
-	if !config.Activate {
+	if !status.EffectiveActivate {
 		if status.Activated || status.ActivateInprogress {
```

Comment:
Don't you need to use EffectiveActivate here as well?

Reply:
It is Activated, not Activate, I think it is correct

Reply:
Oh right